articles/tokens/access-token.md

```diff
@@ -6,15 +6,25 @@ toc: true
 
 ## Overview
 
-The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. Auth0 uses access tokens to protect access to the Auth0 Management API.
+The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` can be any type of token (not necessarily a JSON Web Token) and is meant for the API. The purpose of the `access_token` is to inform the API that the bearer of this token has been authorized to access the API and perform specific actions (as specified by the `scope` that has been granted). The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. 
```

Comment:
I would probably remove this sentence `(not necessarily a JSON Web Token)` since you haven't made the claim that it is a JWT. You could instead type some types in this parenthesis `([JSON Web Tokens](/jwt), opaque strings, whatever-else)`